### ISSUE TITLE
how to start?

### ISSUE THREAD

**Issue by mh25** [TheAirlineProject/tap-desktop#363]
In the latest versions, it looks like the player now starts with no money and you can't take out a loan because you're at the maximum.  Am I missing something?  How does one get started, now?

**Comment by pjank42** [comment]
Is it for all airlines you try with and is the version you are using 042 RC7, since I haven't heard anybody else having that issue?

**Comment by mh25** [comment]
Actually, I was just using a custom airline.  I didn't try it with any
pre-built airline.  I'll give it a try.  I'm using RC7a, but I saw the
issue in RC6 as well.

**Comment by mh25** [comment]
further testing
* problem is still there in RC7b
* using a pre-built airline (in my case, Continental), the problem existed when I changed the home airport
* using another pre-built airline (in this case, Aloha), the problem did not exist when I changed just the time zone and CEO name.

Also, in the history/description for Continental, it lists that it was merged into Delta.  Continental was merged into United in 2012.  Delta tookover Northwest Orient.
